src/lib/tvgCanvasImpl.h

```diff
@@ -114,10 +114,12 @@ struct Canvas::Impl
     {
         if (drawing || paints.count == 0 || !renderer || !renderer->preRender()) return Result::InsufficientCondition;
 
+        bool rendered = false;
         for (auto paint = paints.data; paint < (paints.data + paints.count); ++paint) {
-            if (!(*paint)->pImpl->render(*renderer)) return Result::InsufficientCondition;
+            if ((*paint)->pImpl->render(*renderer)) rendered = true;
         }
 
+        if (!rendered) return Result::InsufficientCondition;
         if (!renderer->postRender()) return Result::InsufficientCondition;
```

Comment:
Could you please one line here?
if (!rendered || !render->postRender()) return ...;